src/main/java/com/kodilla/ecommercee/dto/Group_Entity.java

```diff
@@ -0,0 +1,31 @@
+package com.kodilla.ecommercee.dto;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
```

Comment:
Mozna tutaj użyć Lomboka

Reply:
Klasa Generic Entity , ktora mamy w projekcie to nie jest klasa wzorcowa ? bo na niej sie wzorowalam